sdk/appcenter/src/main/java/com/microsoft/appcenter/persistence/DatabasePersistence.java

```diff
@@ -161,6 +227,16 @@ public void deleteLogs(String group) {
         /* Log. */
         AppCenterLog.debug(LOG_TAG, "Deleting all logs from the Persistence database for " + group);
 
+        /* Delete large payload files */
```

Comment:
Not deleting the directory? Is it intentional?

Reply:
We could but it didn't feel useful as it will be re-created when enabling again. Could add it if you think it's useful.

Reply:
It was just a question. I wondered if you missed it.

Reply:
adding